### PR TITLE
handle options with value 0 properly

### LINKED ISSUE #1284: Clearable icon doesn't show on value = 0
On using the following option, I'm not able to see clearable cross icon on selecting False but it shows in case I select True. 

``` javascript
const ENABLED_FLAG_OPTIONS = [
  { label: 'True', value: 1 },
  { label: 'False', value: 0 },
];

<Select
     value={enabledFlagFilter}
     options={ENABLED_FLAG_OPTIONS}
     onChange={selectedValue => this.setState({ enabledFlagFilter: selectedValue && selectedValue.value })}
/>
```

### PR DESCRIPTION
**Pull request by andreme**
fixes #1284

### CODE REVIEW

**Comment by msaglietto** [comment]
Nice it looks good to me, I agree that 0 it a possible value should not be consider as empty 👍

File: src/Select.js [hunk 1]
```diff
@@ -553,7 +553,7 @@ const Select = React.createClass({
 	 * @param	{Object}		nextProps	- optionally specify the nextProps so the returned array uses the latest configuration
 	 * @returns	{Array}	the value of the select represented in an array
 	 */
-	getValueArray (value, nextProps) {
+	getValueArray (value, nextProps = undefined) {
```
Comment (andreme) [inline src/Select.js]: Yes you're right, but my IDE was warning me that this function was sometimes called with only 1 argument. I thought making it explicit wouldn't hurt.

Comment (malucky) [inline src/Select.js]: This param being optional should be indicated in the jsdoc http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values, which should also stop the IDE warning

**Comment by grumd** [comment]
Is this going to be merged any time soon?

**Comment by hillerstorm** [comment]
No merged PRs since mid-november? Would really like this one to land :/

**Comment by bmv437** [comment]
It would be great to get this merged 👍

**Comment by gwyneplaine** [comment]
Thanks for this @andreme, this has been fixed elsewhere in https://github.com/JedWatson/react-select/pull/1395

**Pull request closed without merging**